contrib/examples/actions/workflows/customfilters/mistral-regex_match.yaml

```diff
@@ -0,0 +1,20 @@
+version: '2.0'
+
+examples.mistral-customfilters-regex_match:
+    description: A workflow for testing regex_match custom filter in mistral
+    type: direct
+    vars:
+        regex_pattern: "([0-9]{3} \\w+ (?:Ave|St|Dr))"
+    input:
+        - input_str
+    output:
+        result_jinja: <% $.result_jinja %>
+        result_yaql: <% $.result_yaql %>
+    tasks:
+
+        task1:
+            action: std.noop
+            publish:
+                result_jinja: '{{ _.input_str | regex_match(_.regex_pattern)}}'
```

Comment:
It also might be worth exploring the idea of modifying Mistral's use of `partial` to specify a keyword arg instead of positional, to be more precise, but I haven't thought that one through yet.